plugins/GSdx/GSDeviceOGL.h

```diff
@@ -412,6 +412,7 @@ class GSDeviceOGL final : public GSDevice
 	uint32 m_msaa;				// Level of Msaa
 	int m_force_texture_clear;
 	int m_mipmap;
+	bool auto_mipmap_full;
```

Comment:
Still missing `bool auto_mipmap_basic;`

Reply:
No need since basic is not called in GSDeviceOGL.cpp